src/views/tickets/tickets.jsx

```diff
@@ -62,19 +63,62 @@ const options = {
   pageButtonRenderer
 };
 
+const expandRow = isSmallScreen => ({
```

Comment:
We can pull out this, the mobileColumns, and the pageButtonRenderer to the components file

Reply:
Unfortunately, we can't, as it uses a hook as a parameter, and hooks have to be called from their components.